File: FirmwareMS/Application/DTOs/FileSendResponseDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.DTOs
{
    public class FileSendResponseDTO : BaseDTO
    {
        public string? UpdateStatus { get; set; }
        public int DeviceId { get; set; }
        public int UserId { get; set; }
        public string? File { get; set; }
        public int CurrPart { get; set; }
        public int TotParts { get; set; }
    }
}

File: FirmwareGO/application/dtos/FileSendResponseDTO.go
package dtos

type FileSendResponseDTO struct {
	BaseDTO              // Embedded
	UpdateStatus *string `json:"updateStatus,omitempty"`
	DeviceID     int     `json:"deviceId"`
	File         *string `json:"file,omitempty"`
	CurrPart     int     `json:"currPart"`
	TotParts     int     `json:"totParts"`
}
